Assets/Mirror/Examples/AdditiveScenes/Scripts/AdditiveNetworkManager.cs

```diff
@@ -26,12 +28,12 @@ public void Started()
 
         IEnumerator LoadSubScenes()
         {
-            if (LogFilter.Debug) Debug.Log("Loading Scenes");
+            if (logger.LogEnabled()) logger.Log("Loading Scenes");
```

Comment:
his if is already performed inside the logger.   

The only reason to use the if, is to avoid calculating `$"... {x}..."`  and `"x" + "y"` In this case we are passing a straight up string,  so simpler is better.  This also improves coverage by having only 1 path.

```suggestion
            logger.Log("Loading Scenes");
```